console/frontend/src/main/frontend/src/app/views/storage/storage.service.ts

```diff
@@ -136,7 +134,7 @@ export class StorageService {
   }
 
   updateTable(): void {
-    this.dtElement?.dtInstance.then((table) => table.draw());
+    // this.dtElement?.dtInstance.then((table) => table.draw());
```

Comment:
I think this is here becouse it is maybe missing this part in the new implementation. But that should be an issue if that's the case. The commend should be removed.

Reply:
This comment should've been removed, its not needed anymore

Reply:
Cleaned up in https://github.com/frankframework/frankframework/pull/7506/commits/0bad7c58386468d1523484a43ffe4babc0db2f95